keyboards/papercranekeyboards/gerald65/readme.md

```diff
@@ -0,0 +1,19 @@
+# Paper Crane Keyboards Gerald65
+
+![gerald65](https://imgur.com/a/G2VuJoD)
+
+A 65% ergonomic acrylic keyboard emphasizing customizability.
+
+* Keyboard Maintainer: [PaperCraneKeyboards](https://github.com/PaperCraneKeyboards)
+* Hardware Supported: Gerald65 PCB
+* Hardware Availability: papercranekeyboards.com
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Added those instructions with specifics for this PCB. Thanks!